packages/ui/src/routes/account/EditAccountPage.tsx

```diff
@@ -21,7 +21,7 @@ const editAccountSchema = yup.object().shape({
         .string()
         .trim()
         .required("Please enter an account name")
-        .max(40, "Account name is too long"),
+        .max(20, "Account name is too long"),
```

Comment:
maybe it's not necessary to limit it that much, we should just make sure that the UI truncates the text everywhere and that we display the full name on hover somehow (could be using "...")

Reply:
Done 💪